Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -31,6 +33,17 @@ class SentryUserFeedbackForm: UIViewController {
         view.backgroundColor = config.theme.background
         initLayout()
         themeElements()
+        
+        NotificationCenter.default.addObserver(forName: UIResponder.keyboardDidShowNotification, object: nil, queue: nil) { note in
+            guard let keyboardValue = note.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue else { return }
+            let keyboardViewEndFrame = self.view.convert(keyboardValue.cgRectValue, from: self.view.window)
+            self.setScrollViewBottomInset(keyboardViewEndFrame.height - self.view.safeAreaInsets.bottom)
+
+        }
+        NotificationCenter.default.addObserver(forName: UIResponder.keyboardWillHideNotification, object: nil, queue: nil) { _ in
+            self.setScrollViewBottomInset(0)
+        }
```

Comment:
h: We need to remove the observer when the view controller disappears.

Also, I would prefer to use a function reference instead of a closure to improve readability.

Reply:
You're right 👍🏻  but we'll only need one or the other. Function-based automatically removes observation.

Reply:
> but we'll only need one or the other

Yes.